presto-main/src/main/java/com/facebook/presto/testing/MaterializedResult.java

```diff
@@ -395,6 +405,17 @@ private static MaterializedRow toTimeZone(MaterializedRow prestoRow, DateTimeZon
         return new MaterializedRow(prestoRow.getPrecision(), values);
     }
 
+    private static ZoneOffset toZoneOffset(TimeZoneKey timeZoneKey)
+    {
+        requireNonNull(timeZoneKey, "timeZoneKey is null");
+        if (Objects.equals("UTC", timeZoneKey.getId())) {
+            return ZoneOffset.UTC;
+        }
+
+        checkArgument(timeZoneKey.getId().matches("[+-]\\d\\d:\\d\\d"), "Not a zone-offset timezone: %s", timeZoneKey);
```

Comment:
Is that granted that we will always have minutes part in the timezone id? Can't we for example get sth like `+1` which is offset based timezone yet would not match the regex?

Reply:
Yes, the time zone ID is normalized for offset-zones in `com.facebook.presto.spi.type.TimeZoneKey#getTimeZoneKey(java.lang.String)`. 
To be double sure, I added such examples to `com.facebook.presto.tests.AbstractTestQueries#testTimeLiterals`